### PR TITLE
Feature: Giant service bell

### LINKED ISSUE #5435: Ding! Add the Service Bell
- [x] The object of this issue is to add the service bell world object to the game, for mappers to place. The service bell sits on a tabletop at the kitchen or bar, and when clicked on, will emit a single, clear, ding.

- [x] Additionally, there is the big service bell, which should have a 0.5% chance of replacing any normal service bell at round start. Instead of the standard ding, it will emit a much larger bell dong! to reflect its bigger size and acoustic capability.

Sprites by Erinexx:
Service Bell: https://cdn.discordapp.com/attachments/409110435900424196/771411832211636315/unknown.png
BIG BELL: https://cdn.discordapp.com/attachments/402986688114065409/731227871321915603/unknown.png

Sounds:
Service Bell: https://puu.sh/GI5HD/6b05344dfc.ogg
BIG BELL: https://puu.sh/GI5Ic/7adc11e7f1.ogg

### PR DESCRIPTION
**Pull request by ghost**
### Purpose
Rarely (0.5% of the time), at round start, the service bell will be replaced with a giant bell that plays a different sound.
image

### Notes:
Closes #5435

### CODE REVIEW

**Comment by ewy1** [comment]
Hey @PerfectTangent! I see you've labelled this with DNM but I don't see why not. Could you clarify?

**Comment by PerfectTangent** [comment]
We currently have a feature freeze in place. See #6275. Even as minor as this is, it technically is a feature, so it's DNM for the time being.

**Comment by ewy1** [comment]
Thanks for the explaination!

File: UnityProject/Assets/Scripts/Items/ServiceBell.cs [hunk 1]
```diff
@@ -5,12 +5,19 @@
 
 namespace Objects
 {
-	public class ServiceBell : Pickupable
+	public class ServiceBell : Pickupable, IServerSpawn
```
Comment (NoooneyDude) [inline UnityProject/Assets/Scripts/Items/ServiceBell.cs:8]: Hmm, I don't see why this should inherit from another component and not `MonoBehaviour`, though it should probably be addressed in another PR.
